tests/unit/split_process_line_test.c

```diff
@@ -117,6 +117,42 @@ static void test_split_line_noelapsed(void)
     free_and_null_strings(field);
 }
 
+static void test_split_line_nocmd(void)
+{
+    /* Collect all test data in one array to make alignments visible: */
+    static const char *lines[] = {
+        /* Indent any continuation lines so that it's clear that's what they are. */
+        /* Use the username field as a test name to confirm tests are in sync. */
+
+        "USER       PID STAT    VSZ  NI   RSS NLWP STIME     ELAPSED     TIME COMMAND",
+        "spacecmd   4 S         0   0     0    1 10:30       54:29 00:00:01  ",
+        "nullcmd    4 S         0   0     0    1 10:30       54:29 00:00:01 ",
```

Comment:
Why does line 1 end with 2 spaces and line 2 end with 1 space? If this is intentional, it deserves a comment.